apps/admin/src/pages/hosts/Info.tsx

```diff
@@ -1,10 +1,139 @@
+import HostApi from '@lib/apis/host/HostApi';
+import { useMutation } from '@tanstack/react-query';
+import usePresignedUrl from '@lib/hooks/usePresignedUrl';
 
 const Info = () => {
+  const [image, setImage] = useState<File | null>(null);
+  const {
+    imageInfo,
+    setImageInfo,
+    uploadImageToS3,
+    getImageExtension,
+    postImageMutation,
+  } = usePresignedUrl();
+  const { displayButtons } = useBottomButton();
+  const [form, onChange, , initForm] = useInputs<UpdateHostRequest>({
+    profileImageKey: '',
+    introduce: '',
+    contactNumber: '',
+    contactEmail: '',
+  });
+
+  useEffect(() => {
+    if (image) {
+      postImageMutation.mutate({
+        hostId: hostId,
+        imageFileExtension: getImageExtension(image!.type),
+      });
+    }
+  }, [image]);
+
+  // profile 수정 api
+  const postEventMutation = useMutation(HostApi.PATCH_HOST_PROFILE, {
+    onSuccess: (data: HostDetailResponse) => {
+      console.log('postHostProfileMutation : ', data);
+    },
+  });
+
+  // 호스트 정보
+  const { pathname } = useLocation();
+  const hostId = pathname.split('/')[2];
+  const { data, status } = useQuery(
+    ['hostDetail'],
+    () => HostApi.GET_HOST_DETAIL(hostId),
+    {
+      onSuccess: (data: HostDetailResponse) => {
+        if (data.profileImageUrl) {
+          setImageInfo({
+            presignedUrl: '',
+            key: getKeyFromUrl(data.profileImageUrl),
+          });
+        }
+        initForm({
+          profileImageKey: '',
+          introduce: data.introduce !== null ? data.introduce : '',
+          contactNumber: data.contactNumber !== null ? data.contactNumber : '',
+          contactEmail: data.contactEmail !== null ? data.contactEmail : '',
+        });
+        console.log('GET_HOST_DETAIL : ', data);
+      },
+    },
+  );
+
+  // 하단부 버튼
+  const buttonClickHandler = () => {
+    // 이미지 post
+    if (image) {
+      uploadImageToS3(imageInfo.presignedUrl, image);
+    }
+    // 호스트 정보 post
+    postEventMutation.mutate({
+      hostId: hostId,
+      payload: { ...form, profileImageKey: imageInfo.key },
+    });
+    console.log(form, image, imageInfo);
+  };
+  useEffect(() => {
+    const buttonDisable = checkButtonDisable(form, imageInfo);
+    displayButtons({
+      type: 'save',
+      firstButtonClickHandler: buttonClickHandler,
+      firstButtonDisable: buttonDisable,
+      isActive: true,
+    });
+  }, [form, image, imageInfo]);
+
   return (
     <>
-      <TempHostIButtonSet />
+      <ListHeader
+        title={'호스트 정보'}
+        size={'listHeader_24'}
+        padding={[32, 0, 40, 0]}
+      />
+      <ContentGrid
+        leftElement={
+          <GridLeftElement
+            hostName={status === 'success' ? data!.name : ''}
+            imageurl={status === 'success' ? data!.profileImageUrl : ''}
+            setImage={setImage}
+          />
+        }
+        rightElement={<GridRightElement onChange={onChange} {...form} />}
+      />
```

Comment:
최대한 기본 HTML 요소 사용법과 비슷하게 가는게 유지보수하기 좋대. 코드를 처음 보는 사람도 예측 가능하도록.

```tsx
const ContentGrid = ({ children, rowGap = 0 }: ContenGridProps) => {
  return <Container rowGap={rowGap}>{children}</Container>;
};
export default ContentGrid;
```

contentGrid 컴포넌트는 그냥 일반 Div와 똑같이 사용하도록 하고,

```suggestion
      <ContentGrid>
        <GridLeftElement
          hostName={status === 'success' ? data!.name : ''}
          imageurl={status === 'success' ? data!.profileImageUrl : ''}
          setImage={setImage}
        />
        <GridRightElement onChange={onChange} {...form} />
      </ContentGrid>
```

그냥 이렇게 children으로 div 요소 두개 넣어주는게 좋을 것 같음!!